src/python/grpcio_tests/tests_py3_only/interop/Dockerfile.client

```diff
@@ -16,13 +17,13 @@ COPY . .
 RUN tools/bazel build -c dbg //src/python/grpcio_tests/tests_py3_only/interop:xds_interop_client
 RUN cp -rL /workdir/bazel-bin/src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client* /artifacts/
 
-FROM phusion/baseimage:master@sha256:e757fe8c7adcb9f798c0eb9dfff31bbf7d91538a1002031d7cdf3e5bf9cf71fc
+FROM python:3.9-slim
+
+RUN apt-get update -y && apt-get upgrade -y
```

Comment:
I think the upgrade should go after the `COPY` statement, otherwise it's possible that the upgrade will not happen regularly due to  layer caching.

Reply:
Done